modules/appnexusBidAdapter.js

```diff
@@ -645,7 +645,7 @@ function newBid(serverBid, rtbBid, bidderRequest) {
       ad: rtbBid.rtb.banner.content
     });
     try {
-      const url = rtbBid.rtb.trackers[0].impression_urls[0];
+      const url = rtbBid.rtb.trackers ? rtbBid.rtb.trackers[0].impression_urls[0] : null;
```

Comment:
Why to create tracking pixel if trackers is null. Can you add this check to whole try block

Reply:
@jaiminpanchal27 Done!